src/load_manager.h

```diff
@@ -147,7 +150,6 @@ class LoadManager {
   bool using_json_data_;
 
   std::shared_ptr<DataLoader> data_loader_;
-  std::unique_ptr<cb::ClientBackend> backend_;
```

Comment:
This wasn't actually used--random cleanup.

Reply:
nice! get that unused variable out of here